packages/page-settings/src/Metadata/NetworkSpecs.tsx

```diff
@@ -106,9 +102,8 @@ function NetworkSpecs ({ chainInfo, className }: Props): React.ReactElement<Prop
     <Table
       className={className}
       empty={t<string>('No open tips')}
```

Comment:
The empty string looks copy-pasted from a different table :-)

Reply:
Changed to `No specification`